iacrcc/iacrcc.cls

```diff
@@ -900,10 +892,12 @@
   \gdef\@IACR@License{#1}%
   \def\@IACR@CCby{CC-by}%
   \ifx\@IACR@License\@IACR@CCby%
-    % doclicense: support for putting documents under a license.
-    \RequirePackage[type={CC},modifier={by},version={4.0}]{doclicense}
+    % doclicense: support for putting documents under a license. Only usable in preamble.
+    \@ifpackageloaded{hyperxmp}{\ClassError{iacrcc}{\string\license\space must be used before \string\begin{document}}{}}
```

Comment:
This checks if hyperxmp is loaded in order to determine if we are in the preamble or not. This might go wrong...
There are ways to check directly if we are in the preample. For example:
`\ifx\@onlypreamble\@notprerr\ClassError{iacrcc}{\string\license\space must be used before \string\begin{document}}{}\fi`

Reply:
Thats a much cleaner way to detect the preamble. I learned something new today.